books/free-programming-books.md

```diff
@@ -1961,8 +1961,8 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Learning React.js: Getting Started and Concepts](https://scotch.io/tutorials/learning-react-getting-started-and-concepts)
 * [React-Bits](https://github.com/vasanthk/react-bits)
 * [React Book, your beginner guide to React](https://github.com/softchris/react-book/) - Chris Noring
-* [React Enlightenment](https://www.gitbook.com/book/frontendmasters/react-enlightenment/details)
-* [React In-depth: An exploration of UI development](https://www.gitbook.com/book/developmentarc/react-indepth/details)
+* [React Enlightenment](https://www.reactenlightenment.com/)
```

Comment:
```suggestion
* [React Enlightenment](https://www.reactenlightenment.com)
```

If you remove the redundant slash in the URL, I think this should be cool.
You can check the failed build below to get more info, that way you can see exactly what's wrong and how to fix it.